srcpkgs/kleopatra/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'kleopatra'
+pkgname=kleopatra
+version=20.12.3
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules kdoctools qt5-qmake qt5-host-tools kcoreaddons gettext kconfig"
+makedepends="kcmutils-devel AppStream kitemmodels-devel gpgmeqt-devel libkleo-devel kmime-devel"
+depends="hicolor-icon-theme libkleo kcmutils kmime gnupg2 gnupg2-scdaemon pinentry-qt"
+short_desc="Certificate Manager and Unified Crypto GUI"
+maintainer="Justin Jagieniak <justin@jagieniak.net>"
+license="GPL-2.0-or-later, LGPL-2.1-or-later, GFDL-1.2-or-later"
+homepage="https://www.kde.org/applications/utilities/kleopatra/"
+distfiles="${KDE_SITE}/release-service/${version}/src/${pkgname}-${version}.tar.xz"
+checksum="bf30f99bc6abd44d512ca8c53a7e81af625db72547a33df5c581101d787feaa0"
+
+pre_check() {
+	export QT_QPA_PLATFORM=offscreen
```

Comment:
```suggestion
	QT_QPA_PLATFORM=offscreen CTEST_OUTPUT_ON_FAILURE=TRUE \
		dbus-run-session ninja -C build test
```
move this to `do_check` instead of `pre_check`

Also you will probably need at least `gnupg2 dbus` in `checkdepends`